integrations/great_expectations/fraud_batch_data_validation.ipynb

```diff
@@ -0,0 +1,3478 @@
+            "03b06f4f6096494f95a0e5fd59086ea2",
+            "c12477beba744563999f369ecaabeb1c",
+            "0898f310459a47fbac23399a51f13e72",
+            "f0824296e88f49ec937a61889a5d1e19",
+            "985470842d604524b0929e0cdefdc14d"
+          ]
+        },
+        "id": "ISVFmoVgC3DP",
+        "outputId": "ded0b525-1f3d-42e6-c112-52bafe6a7d5e"
+      },
+      "id": "ISVFmoVgC3DP",
+      "execution_count": 52,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Validation Report saved successfully, explore a summary at https://c.app.hopsworks.ai:443/p/155/fs/97/fg/555\n"
+          ]
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "text/plain": [
+              "Uploading Dataframe: 0.00% |          | Rows 0/106020 | Elapsed Time: 00:00 | Remaining Time: ?"
+            ],
+            "application/vnd.jupyter.widget-view+json": {
+              "version_major": 2,
+              "version_minor": 0,
+              "model_id": "4172d58379634136889533172366f1cd"
+            }
+          },
+          "metadata": {}
+        },
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Launching offline feature group backfill job...\n",
+            "Backfill Job started successfully, you can follow the progress at \n",
+            "https://c.app.hopsworks.ai/p/155/jobs/named/transactions_fraud_batch_fg_1_offline_fg_backfill/executions\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The \"STRICTLY\" ingestion policy aborts insertion job if the validation fails."
+      ],
+      "metadata": {
+        "id": "jNdOnqj_Mcv0"
+      },
+      "id": "jNdOnqj_Mcv0"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "window_aggs_job, window_aggs_report = window_aggs_fg.insert(window_aggs_df)"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "U-1eNDxWJl1G",
+        "outputId": "e0eabefa-610d-4ee8-e6ac-afdfdd786c1b"
+      },
+      "id": "U-1eNDxWJl1G",
+      "execution_count": 55,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Validation Report saved successfully, explore a summary at https://c.app.hopsworks.ai:443/p/155/fs/97/fg/556\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Explore the Validation Reports in Hopsworks UI"
+      ],
+      "metadata": {
+        "id": "eyfgcXUyNfNp"
+      },
+      "id": "eyfgcXUyNfNp"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Follow the links to explore a summary of the validation reports generated by Great Expectations directly in the Feature Group page!\n",
+        "\n",
```

Comment:
Maybe we can add one screenshot here. so that the notebook is comprehensible also if you just look at it on github/in the docs.

Reply:
Will do that for now. I haven't tried making a small gif but we could show the two tabs (report and result) and the download report button.